templates/web/common/includes/display_data_quality_issues_and_improvement_opportunities.tt.html

```diff
@@ -0,0 +1,25 @@
+<!-- start templates/[% template.name %] -->
+
+[% FOREACH tagtype IN tagtypes %]
+    [% IF tagtype.product_ref_tagtype.defined && tagtype.scalar_product_ref_tagtype > 0 %]
```

Comment:
this check should not be needed in the template: in the Perl code, we should only populate the tagtypes array with non empty values